css-box-model/styles.css

```diff
@@ -0,0 +1,7 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin: 0 16px 0 12px;
```

Comment:
Please don't manually set the top or bottom margin. I suggest avoiding the `margin` shorthand altogether.

---

![image](https://user-images.githubusercontent.com/16143771/167923327-d0baf23c-b955-4ed1-9f5d-9f17a957e277.png)

Reply:
Okay let me try again